Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -8385,6 +8386,16 @@ void HTMLMediaElement::setOverridePreferredDynamicRangeMode(DynamicRangeMode mod
     player->setShouldDisableHDR(shouldDisableHDR());
 }
 
+void HTMLMediaElement::setPlatformDynamicRangeLimit(PlatformDynamicRangeLimit platformDynamicRangeLimit)
+{
+    m_platformDynamicRangeLimit = platformDynamicRangeLimit;
+    if (!m_player)
+        return;
+
+    Ref player = *m_player;
+    player->setPlatformDynamicRangeLimit(platformDynamicRangeLimit);
```

Comment:
Should not this function just be:

```
    if (RefPtr player = m_player)
        player->setPlatformDynamicRangeLimit(platformDynamicRangeLimit);
```

Reply:
I agree, but I was copying the style of the ones above. Happy to be inconsistent in this case. 😄